updates/1/updates.sql

```diff
@@ -1 +1,28 @@
-FILL ME IN
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xaac41EC512808d64625576EDdd580e7Ea40ef8B2", 18, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xaac41EC512808d64625576EDdd580e7Ea40ef8B2", "C", "Gameswap", "GSWAP", 11155251, NULL, "gameswap", "gswap", "0xaac41EC512808d64625576EDdd580e7Ea40ef8B2");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x88ACDd2a6425c3FaAE4Bc9650Fd7E27e0Bebb7aB", 18, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x88ACDd2a6425c3FaAE4Bc9650Fd7E27e0Bebb7aB", "C", "Alchemist", "MIST", 11804891, NULL, "alchemist", "mist", "0x88ACDd2a6425c3FaAE4Bc9650Fd7E27e0Bebb7aB");
+*
+/*Introduce TRB token swap*/UPDATE assets SET swapped_for="_ceth_0x88dF592F8eb5D7Bd38bFeF7dEb0fBc02cf3778a0" WHERE identifier="_ceth_0x0Ba45A8b5d5575935B8158a88C631E9F9C95a2e5";
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x0Ba45A8b5d5575935B8158a88C631E9F9C95a2e5", 18, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x0Ba45A8b5d5575935B8158a88C631E9F9C95a2e5", "C", "Tellor", "TRB", 8265522, NULL, "tellor", "trb", "0x0Ba45A8b5d5575935B8158a88C631E9F9C95a2e5");
```

Comment:
why is the `started`  set to `8265522` here? That's "Mon Apr 06 1970 15:58:42 GMT+0000". Since it's the insertion of the already existing asset it should be `1564671864`.